### ISSUE TITLE
Adding building names to basemap profile

### ISSUE THREAD

**Issue by lokkelvin2** [onthegomap/planetiler#134]
**Describe the bug**
The names of POI/buildings are missing using the default basemap profile.

**Expected behavior**
The output mbtiles should include the name tags of buildings and POIs but they are dropped.

**To Reproduce**
1. Download [Malaysia, Singapore, and Brunei](https://download.geofabrik.de/asia/malaysia-singapore-brunei.html) from geofabrik
2. Run ```java -Xmx1g -jar planetiler.jar --download --osm_path="malaysia-singapore-brunei-old.osm.pbf"```
3. Inspect location [lat: 1.29788, lon: 103.77677](https://www.openstreetmap.org/#map=19/1.29788/103.77677)

**Long description**
I am generating vector tiles for Singapore and have been using [tilemaker ](https://github.com/systemed/tilemaker) prior to planetiler. Using [Way 427077072](https://www.openstreetmap.org/way/427077072) as an example, the building name in OSM is 'Ridge View Tower' and I would expect the output mbtiles to contain the same information. 

In the planetiler mbtiles, however, this name field is missing. Inspecting the planetiler mbtiles using QGIS reveals that planetiler only saves housenumbers and does not save POI/building names.

Here are some screenshots describing the problem:

OSM groundtruth:
![osm1](https://user-images.githubusercontent.com/56050360/158064508-332e460d-e3ee-46ec-988b-b946d46d8cf3.PNG)
Planetiler output:
![planetiler1](https://user-images.githubusercontent.com/56050360/158064417-3d6f212e-0ec9-4972-83f4-1905ce890fa0.PNG)
Tilemaker output:
![tilemaker1](https://user-images.githubusercontent.com/56050360/158064421-1c1fdc05-3a22-4697-bf3b-c78f9aae237c.PNG)

Is there a way I can edit the planetiler profile to match the detailed output of tilemaker/OSM ?

**Environment (please complete the following information):**
- OS: Windows 10
- Java version and distribution: [e.g. Eclipse Temurin 17.0.2]

**Comment by wipfli** [comment]
Inspecting tiles is possible with tileserver-gl

**Comment by wipfli** [comment]
The base profile is derived from OpenMapTiles. Is the house name in OpenMapTiles?

**Comment by wipfli** [comment]
Here is a gist on how to add an OpenStreetMap tag to OpenMapTiles in planetiler https://gist.github.com/wipfli/02ba9081b26c3a0cfae054e5c5402fe7

**Comment by wipfli** [comment]
I am happy to help you hack stuff together. Just ping me in the slack planetiler channel.

**Comment by wipfli** [comment]
https://osmus-slack.herokuapp.com/ generates an invite for the slack channel. Join `#planetiler` once you are in `#general`

**Comment by lokkelvin2** [comment]
> The base profile is derived from OpenMapTiles. Is the house name in OpenMapTiles?

After some digging it seems that OMT does not include buildling names in the schema (ref: https://github.com/openmaptiles/openmaptiles/issues/579 https://github.com/openmaptiles/openmaptiles/issues/241). This means that the planetiler is not wrong in omitting the building names and that my tilemaker script was generating non-compliant OMT tiles :sweat_smile:.

> Here is a gist on how to add an OpenStreetMap tag to OpenMapTiles in planetiler https://gist.github.com/wipfli/02ba9081b26c3a0cfae054e5c5402fe7

This is excellent! Just what I needed. Maybe consider pinning this info somewhere as a I had a hard time piecing together the steps needed to modify the basemap profile to my usecase.

**Comment by wipfli** [comment]
Nice. What would be a good place to post it? How did you find planetiler by the way?

**Comment by msbarry** [comment]
@lokkelvin2 what layer is tilemaker putting them into? `poi` or `building`? I'm trying to keep basemap profile exactly compatible with openmaptiles, but you can add features that you opt into with a command line flag, or in a fork.

Planetiler generates the mapping code in the "generated" package from openmaptiles yaml files so you could change this line to `building: [__any__]`  https://github.com/openmaptiles/openmaptiles/blob/b0e7f7884cc826d3176505ea840bb41f12463c75/layers/poi/mapping.yaml#L375 then regenerate from that. We're eventually planning to make the basemap profile live alongside that config and you won't need to regenerate, but for now this should work.

**Comment by lokkelvin2** [comment]
Tilemaker creates `poi` fields for both `building` and `landuse`. Thanks to your hint, I was able to tweak OMT to get mbtiles near equal with my original Tilemaker script. I'm not entirely sure if I made the OMT yaml file correctly but it seems to output what I need so I'm happy with that. 

I must say that planetiler is awesome and that the blazing speed of planetiler makes iterating through different OMT schemas rather painless.

Also, being able to edit the basemap profile without regeneration would be really useful on machines that do not have a build environment set up.
